eventually/static/src/containers/topicList/TopicItem.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+import FlatButton from 'material-ui/FlatButton';
+import AssignmentList from './AssignmentList';
+
+
+const style = {
+    color: '#455A64',
```

Comment:
you can use colors of material-ui. there is the cool table with it.

Reply:
Hello dear friend )